src/monorepo-workflow-operations.ts

```diff
@@ -75,29 +43,31 @@ function getToday() {
  * possible for a release specification that was created in a previous run to
  * stick around (due to an error). This will ensure that the file is removed
  * first.
+ * @param options.today - The current date.
  * @param options.stdout - A stream that can be used to write to standard out.
  * @param options.stderr - A stream that can be used to write to standard error.
  */
 export async function followMonorepoWorkflow({
   project,
   tempDirectoryPath,
   firstRemovingExistingReleaseSpecification,
+  today,
   stdout,
   stderr,
 }: {
   project: Project;
   tempDirectoryPath: string;
   firstRemovingExistingReleaseSpecification: boolean;
+  today: Date;
   stdout: Pick<WriteStream, 'write'>;
   stderr: Pick<WriteStream, 'write'>;
 }) {
   const releaseSpecificationPath = path.join(tempDirectoryPath, 'RELEASE_SPEC');
 
-  if (firstRemovingExistingReleaseSpecification) {
-    await promisifiedRimraf(releaseSpecificationPath);
```

Comment:
I guess it's fine to leave it around and let it get overwritten. That makes sense. I assume that was the intention here.

Reply:
Right, exactly!